### DISCUSSION TITLE
ビデオチュートリアル

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1821] [Q&A] [answered]
「rakepdf」を生成する前に、画像の入力方法、テキストの配置方法など、ブラウザで表示する方法についてのビデオチュートリアルはありますか？

**Comment by kmuto** [comment, marked as answer]
動画チュートリアルについては、現時点で公式なものは用意していません。
ユーザーによるチュートリアルシリーズがあるようです。 https://www.youtube.com/watch?v=0qoDlEBZeZU&list=PL9F2YWAkk0laTPUR6lFCRZKvS61TuE8oc&index=2
